geomesa-features/geomesa-feature-kryo/src/main/scala/org/locationtech/geomesa/features/kryo/json/KryoJsonSerialization.scala

```diff
@@ -210,7 +210,7 @@ object KryoJsonSerialization extends LazyLogging {
   // write a document without a name - used for the outer-most object which doesn't have a key
   private def writeDocument(out: Output, value: JObject): Unit = {
     val start = out.position()
-    out.setPosition(start + 4) // skip 4 bytes so we have space to go back and write total length
+    out.writeInt(0) // Write int out to prevent oob exceptions so we can resize the buffer at an appropriate time.
```

Comment:
can you leave the original comment about leaving space to go back and write the length?